packages/paste-core/components/date-picker/stories/index.stories.tsx

```diff
@@ -155,9 +205,15 @@ export const DefaultValueDatePicker: React.FC = (props) => {
   const uidHT = useUID();
   return (
     <>
-      <Label htmlFor={uidDP}>When should we reach out to you for an update?</Label>
-      <DatePicker id={uidDP} aria-describedby={uidHT} defaultValue="2020-03-18" {...props} />
-      <HelpText id={uidHT}>Change the date above if necessary.</HelpText>
+      <Label htmlFor={uidDP}>
+        When was Planned Parenthood founded in the US (under the name American Birth Control League)?
```

Comment:
NIT: Should this be "Planned Parenthood (PPFA)" so the acronym in the anchor text is more clear?